SwiftMonkey/Monkey.swift

```diff
@@ -147,12 +147,13 @@ public class Monkey {
         }
     }
 
-    /// Generate random events forever, or until the app crashes.
-    public func monkeyAround() {
-        while true {
+    /// Generate random events or fixed-interval events based on a specific duration or infinitely
+    public func monkeyAround(forDuration duration: TimeInterval) {
```

Comment:
Could you please make it an additional method so that we keep the parameterless `monkeyAround()` still available? I can see from the commit summary that:
> This doesn’t remove the infinite running of monkeyAround() ...

But I'm afraid it does. When you try to compile the tests for the provided sample app the compilation will fail.

Reply:
True true, .infinitely can be used but that will mean they will have to update their test(s), sorry, I didn't consider that 🙈 okay, I will make a fix later today and will update the PR.

Reply:
Ah I totally forgot about the `.infinity` value! With that the solution is even more simple:
```swift
public func monkeyAround(forDuration duration: TimeInterval = .infinity) {
    ...
}
```
Great idea @grycyk 👍